go/src/koding/klientctl/main.go

```diff
@@ -863,16 +884,6 @@ error opening: %s
 			},
 		}},
 	}, {
-		Name:   "sync",
```

Comment:
👍 , Although, I think that `sync` command is only specific to mounts and should not be put in the global namespace. It's easier to add it later if frequently requested, but will be much harder to remove without breaking changes if it causes users' confusion.